.gitignore

```diff
@@ -1 +1,2 @@
 _site
+.DS_Store
```

Comment:
(Interesting – github apparently only shows that warning in whole-file-view mode, not when looking at the inline pull request comment here.)